test/e2e/create_test.go

```diff
@@ -58,6 +58,18 @@ func testWindowsNodeCreation(t *testing.T) {
 	// Create a private key secret with the known private key.
 	require.NoError(t, testCtx.createPrivateKeySecret(true), "could not create known private key secret")
 
+	// in case platform=none is detected just wait for Windows worker node to become available
+	if testCtx.CloudProvider.GetType() == config.NonePlatformType {
+		// TODO: setPowerShellDefaultShell for given Windows instance(s)
+		// TODO: validate windows-instances configMap
```

Comment:
Nice TODOs. I suggest flipping their order when implementing them however (fail faster).

Reply:
Are you planning to address the TODOs here or in a follow up PR?

Reply:
No longer need them, I re-worked this based on Sebastian's [comment](https://github.com/openshift/windows-machine-config-operator/pull/858#discussion_r780316359).